recipes/mu-repo/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "mu_repo" %}
```

Comment:
@nicoddemus, @ocefpaf already took care of the rename: https://github.com/conda-forge/staged-recipes/pull/2726